File: csharp/SEP3WebAPI/Mediator/Messages/OrderMessage.cs
using System.Collections.Generic;
using SEP3Library.Models;

namespace SEP3WebAPI.Mediator.Messages {
    public class OrderMessage : Message {
        public int Index { get; set; }
        public Order Order { get; set; }
        public IList<Order> Orders { get; set; }
        public int CustomerId { get; set; }
        public string Status { get; set; }
    }
}

File: java/src/main/java/mediator/message/OrderMessage.java
package mediator.message;

import model.Order;

import java.util.ArrayList;
import java.util.List;

public class OrderMessage extends Message {
    private int index;
    private List<Order> orders;
    private int customerId;
    private String status;

    public OrderMessage(String service, String type) {
        super(service, type);
        orders = new ArrayList<>();
    }

    public int getIndex() {
        return index;
    }

    public void setIndex(int index) {
        this.index = index;
    }

    public List<Order> getOrders() {
        return orders;
    }

    public void setOrders(List<Order> orders) {
        this.orders = orders;
    }

    public int getCustomerId() {
        return customerId;
    }

    public void setCustomerId(int customerId) {
        this.customerId = customerId;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}
